mod_reforged/hooks/skills/actives/gore_skill.nut

```diff
@@ -0,0 +1,21 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/gore_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
+		this.m.Description = "Charge forward with your unnatural spiritual energy and gore your victims on your massive horns.";
```

Comment:
I think we can remove some yours from this sentence.

"Charge forward with unnatural spiritual energy and gore the enemy on your massive horns."